packages/manager/modules/nasha/src/dashboard/routing.js

```diff
@@ -28,6 +28,10 @@ export default /* @ngInject */ ($stateProvider) => {
       resolve: {
         hideBreadcrumb: () => true,
       },
+      translations: {
+        value: ['.', '../add'],
+        format: 'json',
+      },
```

Comment:
Draft PR has I'm looking to check if it can't be done by `$translate.refresh()`.